### ISSUE TITLE
Added `utterances-bot` to private repo

### ISSUE THREAD

**Issue by femtomc** [utterance/utterances#559]
Is there an automated way to do this? I have a request out but not sure what to do now.

**Comment by nschonni** [comment]
Duplicate of https://github.com/utterance/utterances/issues/203

**Comment by femtomc** [comment]
Yes indeed. But it seemed that issue was never resolved, as of January of last year? Is this option a thing yet?

**Comment by femtomc** [comment]
Re -- in a related closed issue: https://github.com/utterance/utterances/issues/256 it seems like if someone accepts a private repo request to add `utterances-bot` as a collaborator, this should work.
